content/telegraf/v1.22/release-notes-changelog.md

```diff
@@ -9,7 +9,44 @@ menu:
     name: Release notes
     weight: 60
 ---
-## v1.22.0 [2022-4-6]
+## v1.22.2 [2022-4-26]
+
+- Allow Makefile to work on Windows.
+
+## Input plugin updates
+- Aerospike (`aerospike`): Fix statistics query bug.
+- Aliyun CMS (`aliyuncms`): Ensure metrics accept array.
+- Cisco Telemetry MDT (`cisco_telemetry_mdt`):
+  - Align the default value for message size.
+  - Remove overly verbose info message.
+- GNMI (`gnmi`):
+  - Add mutex to lookup map.
+  - Use sprint to cast to strings.
+- Consul agent (`consul_agent`): Use correct auth token.
+- MySQL (`mysql`): Add `mariadb_dialect` to address the MariaDB differences in `INNODB_METRICS`.
+- SMART (`smart`): Correctly parse various numeric forms
+- Prometheus (`prometheus`): Moved from watcher to informer.
+
+## Output plugin updates
+- InfluxDB v2 (`influxdb_v2`): Improve error message.
+- Allow zero outputs when using `test-wait` parameter.
```

Comment:
This isn't an Output plugin enhancement - you could add this to the top under "Allow makefile..."